lang/nl.js

```diff
@@ -20,7 +20,7 @@ export default {
 	"components.filter.clearDescription": "Filters wissen voor {filterName}",
 	"components.filter.loading": "Laden van filters",
 	"components.filter.filterCountDescription": "{number, plural, =0 {Geen filters toegepast.} one {{number} filter toegepast.} other {{number} filters toegepast.}}",
-	"components.filter.filters": "Filters", // mfv-translated
+	"components.filter.filters": "filters",
```

Comment:
The capitalization in this translation was changed. I changed it back to the original `Filters`. Is that an appropriate change?

Reply:
Seems reasonable, but a bit suspicious that it was changed...